ecs-agent/daemonimages/csidriver/driver/node.go

```diff
@@ -69,7 +70,8 @@ type nodeService struct {
 }
 
 func (d *nodeService) NodeStageVolume(ctx context.Context, req *csi.NodeStageVolumeRequest) (*csi.NodeStageVolumeResponse, error) {
-	klog.V(4).InfoS("NodeStageVolume: called", "args", *req)
```

Comment:
Q: Why are we leaving out `V(4)`?

Reply:
We added the V(4) to all logging statements.